openmetadata-ui/src/main/resources/ui/src/pages/AlertDataInsightReportPage/AlertDataInsightReportPage.tsx

```diff
@@ -102,14 +116,22 @@ const AlertDataInsightReportPage = () => {
               subHeader: dataInsightAlert?.description ?? '',
             }}
           />
-
-          <Link
-            data-testid="edit-button"
-            to={`${EDIT_DATA_INSIGHT_REPORT_PATH}/${dataInsightAlert?.id}`}>
-            <Button icon={<Icon component={IconEdit} size={12} />}>
-              {t('label.edit')}
+          <Space size={16}>
+            <Link
+              data-testid="edit-button"
+              to={`${EDIT_DATA_INSIGHT_REPORT_PATH}/${dataInsightAlert?.id}`}>
+              <Button icon={<Icon component={IconEdit} size={12} />}>
+                {t('label.edit')}
+              </Button>
+            </Link>
+            <Button
+              data-testid="send-now-button"
+              icon={<Icon component={IconSend} size={12} />}
+              loading={isSendingReport}
+              onClick={handleSendDataInsightReport}>
+              {t('label.send-now')}
```

Comment:
What if there's no `dataInsightAlert` or there will be always one?
Also styling for button is not same as what we have, Can we loose icon here? Looks unwanted

Reply:
We have system-generated `dataInsightAlert`  so there will always be one.
Button styling is the same as we have on other alert pages.